tex2pdf_service/tex2pdf/tex_to_pdf_converters.py

```diff
@@ -78,6 +78,52 @@ def produce_pdf(self, tex_file: str, work_dir: str, in_dir: str, out_dir: str) -
         """Produce PDF from the given tex file. Return the outcome dict."""
         pass
 
+    @abstractmethod
+    def _base_runner(self, step: str, tex_file:str, work_dir: str, in_dir: str, out_dir: str) -> dict:
```

Comment:
suggest _latexen_run in lie of _base_runner, and change _lapex_run and _pdflatex_run to be _latexen_run.

And get rid of _base_runner overriding.